nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -624,7 +624,7 @@ class ReconAllInputSpec(CommandLineInputSpec):
                         desc="Number of processors to use in parallel")
     subjects_dir = Directory(exists=True, argstr='-sd %s', hash_files=False,
                              desc='path to subjects directory', genfile=True)
-    flags = traits.Str(argstr='%s', desc='additional parameters')
+    flags = traits.Either(traits.Str, traits.List, argstr='%s', desc='additional parameters')
```

Comment:
this is what we use `InputMultiPath` for . so the above would reduce to `InputMultiPath(Str, argstr=...)`